cypress/integration/article_spec.js

```diff
@@ -0,0 +1,22 @@
+describe('News Article', function() {
+var url = 'http://localhost:3000/';
+
+    it('should render a headline', function(){
+        cy.visit(url)
+        cy.url().should('equal', url)
+        cy.get('h1').should('contain','Article Headline')
+    })
+
+    it('should render a title', function(){
+    	cy.visit(url)
+    	cy.title().should('eq', 'Welcome to the Afterparty')
+
+    })
+
+    it('should display 200/OK', function(){
```

Comment:
Do the tests run sequentially? If so, this should be first.

Also, this should be an arrow function.